LoopKitUI/Views/Settings Editors/TherapySettingsViewModel.swift

```diff
@@ -54,17 +50,25 @@ public class TherapySettingsViewModel: ObservableObject {
         self.chartColors = chartColors
         self.didSave = didSave
     }
-    
+
+    var therapySettingsGlucoseUnit: HKUnit {
+        therapySettings.glucoseUnit ?? .milligramsPerDeciliter
```

Comment:
[nit] Shouldn't `therapySettings` always have a `glucoseUnit`? If not, the `therapySettings` values don't make sense.